lightly_studio/src/lightly_studio/utils/download.py

```diff
@@ -0,0 +1,82 @@
+"""Module for downloading example datasets from the web."""
+
+import os
+import shutil
+import zipfile
+from pathlib import Path
+
+import requests
+from tqdm import tqdm
+
+# URL to download the main branch of the repo as a zip
+ZIP_URL = "https://github.com/lightly-ai/dataset_examples/archive/refs/heads/main.zip"
+# name of the folder inside the zip
+REPO_DIR_IN_ZIP = "dataset_examples-main"
+
+
+def download_example_dataset(target_dir: str = "dataset_examples", force: bool = False) -> str:
```

Comment:
I would name the variable differently:
- The dir the dataset is download to is the target for this function, but it is the input to other functions. Thus I would not name it target.
- It should be specified what is forced.

```suggestion
def download_example_dataset(download_dir: str = "dataset_examples", force_redownload: bool = False) -> str:
```